packages/mux-player/src/helpers.ts

```diff
@@ -112,3 +112,99 @@ export const isInLiveWindow = (el: MuxPlayerElement) => {
   }
   return false;
 };
+
+export class AttributeTokenList {
```

Comment:
question(non-blocker): Any reason we didn't build on top of `DOMTokenList`? (https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList)

Reply:
It isn't possible to instantiate the built-in DOMTokenList, unfortunately.

Reply:
well that's simultaneously a good reason and a bad reason.

Reply:
yeah, I even just tried if I could extend it, but it doesn't let you because DOMTokenList is only an interface, unfortuantely.

Reply:
ah if that's the case (I again don't think this should be a blocker) but @luwes might be worth trying to add `implements DOMTokenList` and at least see if there are any discrepancies between your impl and the "official" interface

Here's the def:
```ts
interface DOMTokenList {
    [Symbol.iterator](): IterableIterator<string>;
    entries(): IterableIterator<[number, string]>;
    keys(): IterableIterator<number>;
    values(): IterableIterator<string>;
}
```